src/stories/BlockTag.stories.js

```diff
@@ -19,8 +19,7 @@ const ftvamock = {
 
 // ftva site
 const ftvamockwLinkLabel = {
-  label: '<a href="/">label with html link tags</a>',
```

Comment:
why has this been removed? Is UX saying now that they will not pass <a> tags as labels?

Reply:
It is not a link.